kore/src/Kore/Step/Simplification/AndTerms.hs

```diff
@@ -535,6 +537,30 @@ andEqualsFunctions =
 
     lift = pure . transformerLiftOld
     liftE = lift . toExpanded
+    liftE0
```

Comment:
Why do these definitions need to be exploded, unlike the others?

Reply:
To be entirely honest, the naming conventions in this where clause are not clear to me at all. Maybe we can discuss this when I present this PR.